src/v2i-hub/CARMAStreetsPlugin/src/CARMAStreetsPlugin.h

```diff
@@ -155,6 +161,14 @@ class CARMAStreetsPlugin: public PluginClient {
 	 * @brief Intersection Id for intersection
 	 */
 	std::string _intersectionId = "UNSET";
+	/**
+	 * @brief Status label for SRM messages skipped due to errors.
+	 */
+	const char* Key_SRMToJSONSkipped = "Failed to convert J2735 SRM To JSON due to errors.";
```

Comment:
Please make this align in naming with the other status keys in this class

Reply:
What exact name do you suggest?

Reply:
https://github.com/usdot-fhwa-OPS/V2X-Hub/pull/518/commits/74637787cb7971195afe8922725894f2e01feac6